config.default.json

```diff
@@ -11,9 +11,9 @@
     "MAX_SIZE": 10000,
     "MAX_AGE_MS": 60000
   },
-  "WORKERS_COUNT": 10,
+  "WORKERS_COUNT": 5,
   "REQUEST_QUEUE_LENGTH": 2,
-  "QUEUE_LENGTH_LOG_FREQUENCY_MINUTES": 15,
+  "QUEUE_LENGTH_LOG_FREQUENCY_MINUTES": 1,
```

Comment:
is this intentional?

Reply:
Yes, it's generating a debug log so for almost everyone it would not have any effect!
I find that 15 mins is too infrequent especially if you are looking at the snyk-monitor running in the cluster, it takes a long time to debug the queue growth